packages/vats/test/bootstrapTests/test-liquidation.js

```diff
@@ -0,0 +1,592 @@
+  Collateral: {
+    value: scale6(Collateral),
+  },
+  Bid: {
+    value: scale6(Bid),
+  },
+});
+
+const makeDefaultTestContext = async t => {
+  console.time('DefaultTestContext');
+  const swingsetTestKit = await makeSwingsetTestKit(t, 'bundles/vaults', {
+    configSpecifier: '@agoric/vats/decentral-main-vaults-config.json',
+  });
+
+  const { runUtils, storage } = swingsetTestKit;
+  console.timeLog('DefaultTestContext', 'swingsetTestKit');
+  const { EV } = runUtils;
+
+  // Wait for ATOM to make it into agoricNames
+  await EV.vat('bootstrap').consumeItem('vaultFactoryKit');
+  console.timeLog('DefaultTestContext', 'vaultFactoryKit');
+
+  // has to be late enough for agoricNames data to have been published
+  const agoricNamesRemotes = makeAgoricNamesRemotesFromFakeStorage(
+    swingsetTestKit.storage,
+  );
+  agoricNamesRemotes.brand.ATOM || Fail`ATOM missing from agoricNames`;
+  console.timeLog('DefaultTestContext', 'agoricNamesRemotes');
+
+  const walletFactoryDriver = await makeWalletFactoryDriver(
+    runUtils,
+    storage,
+    agoricNamesRemotes,
+  );
+  console.timeLog('DefaultTestContext', 'walletFactoryDriver');
+
+  const governanceDriver = await makeGovernanceDriver(
+    swingsetTestKit,
+    agoricNamesRemotes,
+    walletFactoryDriver,
+    // TODO read from the config file
+    [
+      'agoric1gx9uu7y6c90rqruhesae2t7c2vlw4uyyxlqxrx',
+      'agoric1d4228cvelf8tj65f4h7n2td90sscavln2283h5',
+      'agoric14543m33dr28x7qhwc558hzlj9szwhzwzpcmw6a',
+    ],
+  );
+  console.timeLog('DefaultTestContext', 'governanceDriver');
+
+  const priceFeedDriver = await makePriceFeedDriver(
+    storage,
+    agoricNamesRemotes,
+    walletFactoryDriver,
+    // TODO read from the config file
+    [
+      'agoric1krunjcqfrf7la48zrvdfeeqtls5r00ep68mzkr',
+      'agoric19uscwxdac6cf6z7d5e26e0jm0lgwstc47cpll8',
+      'agoric144rrhh4m09mh7aaffhm6xy223ym76gve2x7y78',
+      'agoric19d6gnr9fyp6hev4tlrg87zjrzsd5gzr5qlfq2p',
+      'agoric1n4fcxsnkxe4gj6e24naec99hzmc4pjfdccy5nj',
+    ],
+  );
+
+  console.timeLog('DefaultTestContext', 'priceFeedDriver');
+
+  console.timeEnd('DefaultTestContext');
+
+  return {
+    ...swingsetTestKit,
+    agoricNamesRemotes,
+    governanceDriver,
+    priceFeedDriver,
+    walletFactoryDriver,
+  };
+};
+
+test.before(async t => {
+  t.context = await makeDefaultTestContext(t);
+});
+test.after.always(t => {
+  return t.context.shutdown && t.context.shutdown();
+});
+
+// Reference: Flow 1 from https://github.com/Agoric/agoric-sdk/issues/7123
+test.serial('scenario: Flow 1', async t => {
+  const {
+    advanceTimeBy,
+    agoricNamesRemotes,
+    governanceDriver,
+    readLatest,
+    priceFeedDriver,
+    advanceTimeTo,
+    walletFactoryDriver,
+  } = t.context;
+
+  // price feed logic treats zero time as "unset" so advance to nonzero
+  await advanceTimeBy(1, 'seconds');
+
+  await priceFeedDriver.setPrice(12.34);
+
+  // raise the VaultFactory DebtLimit
```

Comment:
consider changing comments of this sort into `t.log()` (or console.log)

Of all the things I would want to see printed, these are way up there.

Reply:
I see `console.log('step 1 of 10');` below... maybe the steps here are less interesting / relevant to the test?

Reply:
the problem with `t.log()` is it all outputs at the end, making it hard to debug the sequencing. I used console so I could see what was happening at each step